scripts/create_pelorus_operator.sh

```diff
@@ -130,7 +130,11 @@ pushd "${destination_dir}" || exit 2
     echo "INFO: Creating operator init files"
     operator-sdk init --plugins=helm --domain "${OPERATOR_PROJECT_DOMAIN}" --project-name "${OPERATOR_PROJECT_NAME}" || exit 2
     echo "INFO: Creating api"
+    # https://github.com/operator-framework/operator-sdk/issues/4689#issuecomment-992967172
```

Comment:
Actually we have to open an issue there, this comment may have led us to the problem

The problem is: when you create a helm api with dependencies, it copies the dependency as `tgz` file, instead of coping the files one by one, then it breaks saying the files/folder does not exist, but they are there, but were not unpacked...